documentation/topics.ditamap

```diff
@@ -3,7 +3,14 @@
 <map id="detect-user-guide" xml:lang="en-US">
   <title>Synopsys Detect User Guide</title>
   <topicmeta>
-    <shortdesc>Synopsys Detect 8.4.0</shortdesc>
+	<shortdesc>Synopsys Detect</shortdesc>
+        <prodinfo>
+            <prodname>Synopsys Detect</prodname>
+            <vrmlist>
+                <vrm version="8.4.0"/>
+            </vrmlist>
+        </prodinfo>
+
     <!-- zoomin bundle name -->
```

Comment:
I'm not sure this change will have the desired effect. If I understood what Mark told me correctly, I believe it is the detect.ditamap that matters during the import into Community. detect.ditamap gets generated during the build from documentation/src/main/resources/templates/ditamap.ftl (so that the solution name and current version can be written to it automatically). You may want to make changes to that file.

Reply:
Hey Steve, to show the version in the doc title (Particularly useful for anyone downloading the PDF), I'll make a change in the template file you mention, so thank you for that. For the secondary plan of "book" versioning that will come into play with the above. I am theoretically going to use it to point plugin users to the hosted topic(s) for the version of Detect their plugin relates to once we are hosting multiple versions. (Still theoretical)

Reply:
Oh interesting. If it helps, it shouldn't be too hard to template-ize more files, to get the build to plugin the version in more places.

Reply:
FYI - changes made in the ditamap.ftl to provide the version in the document title. Thanks.